tests/Integration/Integrations/Memcached/MemcachedTest.php

```diff
@@ -42,7 +43,7 @@ public function testAdd()
         $this->assertSpans($traces, [
             SpanAssertion::build('Memcached.add', 'memcached', 'memcached', 'add')
                 ->withExactTags(array_merge(self::baseTags(), [
-                    'memcached.query' => 'add key',
+                    'memcached.query' => 'add ' . Obfuscation::toObfuscatedString('key'),
```

Comment:
Nit-pick 😄.
This is not a blocker for me and will approve the PR, but wouldn't you agree that we should write `'?'` here in place of `Obfuscation::toObfuscatedString('key')`? What if we change the glue at a certain point in the utility class? We would like to know that the user expectation is not matched anymore. 
I am always in favor of setting the "purest" expectation as possible in tests. But again, not a blocker to me!

Reply:
I agree! Although it technically makes the tests more brittle, I think that's what we want in this case. Also, the tests would be way easier to read. I'll revert this. :)